src/content/general/tools/3dsmax/gmax/page.yml

```diff
@@ -0,0 +1,10 @@
+title:
+  en: Gmax
+img: gmax.png
+imgCaption:
+  en: Gmax's user interface is nearly identical to 3ds Max's.
+info:
+  en: |
+    * [Wikipedia](https://en.wikipedia.org/wiki/Gmax)
+    * [Download (TurboSquid)](https://www.turbosquid.com/gmax)
+toolName: Gmax
```

Comment:
From here on out the diff gets pretty repetitive. I wrote a script to mass convert all YAML page metadata from the old format to the new.